### PR TITLE
Log redact on TiFlash

### PR DESCRIPTION
**Pull request by JaySon-Huang**
Signed-off-by: JaySon-Huang <tshent@qq.com>

### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Add redact log configurations for TiFlash.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s): https://github.com/pingcap/docs-cn/pull/5094

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by JaySon-Huang** [comment]
The manual cherry-pick for the release-4.0 branch: https://github.com/pingcap/docs-cn/pull/5688

**Review by TisonKun — APPROVED** [review]
/LGTM

File: log-redaction.md [hunk 1]
```diff
@@ -42,3 +42,7 @@ TiKV 侧的日志脱敏需要将 [`security.redact-info-log`](/tikv-configuratio
 ## PD 组件日志脱敏
 
 PD 侧的日志脱敏需要将 [`security.redact-info-log`](/pd-configuration-file.md#redact-info-log-从-v500-rc-版本开始引入) 的值设为 `true`。该配置项值默认为 `false`，即关闭脱敏。
+
+## TiFlash 组件日志脱敏
+
+TiFlash 侧的日志脱敏需要将 tiflash-server 里 [`security.redact_info_log`](/tiflash/tiflash-configuration.md#配置文件-tiflashtoml) 的值以及 tiflash-learner 里的 [`security.redact-info-log`](/tiflash/tiflash-configuration.md#配置文件-tiflash-learnertoml) 的值均设为 `true`。两配置项默认值均为 `false`，即关闭脱敏。
```
Comment (TomShawn) [inline log-redaction.md:48]:
```suggestion
TiFlash 侧的日志脱敏需要将 tiflash-server 中 [`security.redact_info_log`](/tiflash/tiflash-configuration.md#配置文件-tiflashtoml) 配置项的值以及 tiflash-learner 中 [`security.redact-info-log`](/tiflash/tiflash-configuration.md#配置文件-tiflash-learnertoml) 配置项的值均设为 `true`。两配置项默认值均为 `false`，即关闭脱敏。
```

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**